package.json

```diff
@@ -21,7 +21,7 @@
         "jshint": ">=1.1.0",
         "mocha": ">=1.9.0",
         "mocha-istanbul": ">=0.2.0",
```

Comment:
I think we can get rid of `istanbul` and `mocha-istanbul`, or do we have code coverage that I'm not aware of?